modules/vmseries/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.2, < 2.0"
+  required_version = ">= 1.5, < 2.0"
```

Comment:
The same as in previous one - shouldn't we use `required_version = ">= 1.3, < 2.0"` ?